### ISSUE TITLE
Enable Top Shelf items on AppleTV

### ISSUE THREAD

**Issue by smidley** [jellyfin/Swiftfin#335]
AppleTV apps that are in the top shelf row normally have media items that show on the main part of the screen.  It would be great if we could have Jellyfin show media items that are selectable here as well.

Right now when the Jellyfin application is in the top shelf and highlighted, it just shows the Jellyfin logo in the background.

This feature should also try to accommodate parental controls settings that are set in the AppleTV settings.

Top Shelf example:
![](https://developer.apple.com/design/human-interface-guidelines/tvos/images/tvOS-home-screen_2x.png)

Top Shelf Guidelines:
https://developer.apple.com/design/human-interface-guidelines/tvos/overview/top-shelf/

**Comment by LePips** [comment]
This indeed was started by somebody else, but it was so long ago that it will have to be re-done.

https://github.com/jellyfin/Swiftfin/pull/224

**Comment by stevebyatt10** [comment]
Has the concerns about the persistent store being a shared container been addressed yet?

**Comment by LePips** [comment]
No, but I'm sorry to say you're going to have to figure that out with CoreStore. I won't have the time for a while to look at this stuff while I focus on other things.

**Comment by stevebyatt10** [comment]
I will look into solving the CoreStore issue and re-implementing what i had before.
